src/js/view/controls/rightclick.js

```diff
@@ -104,7 +104,7 @@ export default class RightClick {
     }
 
     hideMenu(evt) {
-        if (evt && this.el.contains(evt.target)) {
+        if (evt && this.el && this.el && this.el.contains(evt.target)) {
```

Comment:
It looks like `this.el && this.el &&` was repeated by accident.

Reply:
Sorry about that, don't know how it snuck in there twice 😄